### PR TITLE
Daily tests

### PR DESCRIPTION
**Pull request by NissimHadar**
# Description
1.  Adds a method to the Test class that maximizes the window.  This is needed so that snapshots taken with the primary camera will have a constant size.
2.  Added options to AutoTester. to show and hide the Windows task bar.
# Testing
1.  Make sure the local sandbox is running
2.  Run interface from the command line with any test script
     `interface.exe --testScript D:\GitHub\hifi\scripts\defaultScripts.js`
3.  Resize the Interface window to some size (e.g. half the screen size).
4.  Open the Scripting Console (CTRL+ALT+J)
5.  Enter `Test.showMaximized();` and verify that the window becomes full screen

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10153/HighFidelity-Beta-PR13247-5fd8cd71c60fbd9e51ffcc70eb1c1c60fbaa179b.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10153/HighFidelity-Beta-PR13247-5fd8cd71c60fbd9e51ffcc70eb1c1c60fbaa179b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10153/HighFidelity-Beta-Interface-PR13247-5fd8cd71c60fbd9e51ffcc70eb1c1c60fbaa179b.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10153/HighFidelity-Beta-PR13247-5fd8cd71c60fbd9e51ffcc70eb1c1c60fbaa179b.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10153/HighFidelity-Beta-Interface-PR13247-5fd8cd71c60fbd9e51ffcc70eb1c1c60fbaa179b.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10157/HighFidelity-Beta-PR13247-658c0e329fab2f78f43b1a340c5bb35b656ab83a.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10157/HighFidelity-Beta-PR13247-658c0e329fab2f78f43b1a340c5bb35b656ab83a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10157/HighFidelity-Beta-Interface-PR13247-658c0e329fab2f78f43b1a340c5bb35b656ab83a.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10157/HighFidelity-Beta-PR13247-658c0e329fab2f78f43b1a340c5bb35b656ab83a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10157/HighFidelity-Beta-Interface-PR13247-658c0e329fab2f78f43b1a340c5bb35b656ab83a.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10172/HighFidelity-Beta-PR13247-0af1599b532c45fbb849fbf6c127855fe7b8a567.apk

File: interface/src/scripting/TestScriptingInterface.h [hunk 1]
```diff
@@ -27,72 +27,116 @@ public slots:
...
     /**jsdoc
     * Waits for all pending downloads, parsing and texture transfers to be complete
+    * @function Test.waitIdle
     */
     void waitIdle();
 
+    /**jsdoc
+    * Waits for establishment of connection to server
+    * @function Test.waitForConnection
+    */
```
Comment (MiladNazeri) [inline interface/src/scripting/TestScriptingInterface.h]: Can you add the default param here?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10172/HighFidelity-Beta-PR13247-0af1599b532c45fbb849fbf6c127855fe7b8a567.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10172/HighFidelity-Beta-Interface-PR13247-0af1599b532c45fbb849fbf6c127855fe7b8a567.dmg

File: interface/src/scripting/TestScriptingInterface.h [hunk 2]
```diff
@@ -27,72 +27,116 @@ public slots:
...
     void endTraceEvent(QString name);
 
     /**jsdoc
      * Write detailed timing stats of next physics stepSimulation() to filename
+     * @function Test.savePhysicsSimulationStats
+     * @param {string} filename - Name of file to save to
      */
     void savePhysicsSimulationStats(QString filename);
 
     Q_INVOKABLE void profileRange(const QString& name, QScriptValue function);
```
[the remark this answers is not in the document]
Comment (NissimHadar) [reply interface/src/scripting/TestScriptingInterface.h]: Updated

**Comment by NissimHadar** [comment]
Done.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10179/HighFidelity-Beta-PR13247-5fb7ec7b72f675b97cd3729c93ac634aefa703ab.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10179/HighFidelity-Beta-PR13247-5fb7ec7b72f675b97cd3729c93ac634aefa703ab.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10179/HighFidelity-Beta-Interface-PR13247-5fb7ec7b72f675b97cd3729c93ac634aefa703ab.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10179/HighFidelity-Beta-PR13247-5fb7ec7b72f675b97cd3729c93ac634aefa703ab.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10179/HighFidelity-Beta-Interface-PR13247-5fb7ec7b72f675b97cd3729c93ac634aefa703ab.exe

File: interface/src/scripting/TestScriptingInterface.h [hunk 3]
```diff
@@ -27,72 +27,116 @@ public slots:
...
 
     /**jsdoc
     * Clear all caches (menu command Reload Content)
+    * @function Test.clearCaches
     */
     void clearCaches();
 
     /**jsdoc
     * Save a JSON object to a file in the test results location
+    * @function Test.saveObject
```
[the remark this answers is not in the document]
Comment (NissimHadar) [reply interface/src/scripting/TestScriptingInterface.h]: Done.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10250/HighFidelity-Beta-PR13247-bea21e3.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10250/HighFidelity-Beta-PR13247-bea21e3.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10250/HighFidelity-Beta-Interface-PR13247-bea21e3.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10250/HighFidelity-Beta-PR13247-bea21e3.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10250/HighFidelity-Beta-Interface-PR13247-bea21e3.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Screen maximized.

**Pull request merged**